ITEM.tsv

```diff
@@ -8523,261 +8523,261 @@ ITEM_20151102_008522	Unsent Letters
+ITEM_20151224_008535	Dragonfly Armband
+ITEM_20151224_008536	Armband with Feelings
+ITEM_20151224_008537	No_hair
+ITEM_20151224_008538	Terius
+ITEM_20151224_008539	Hair [Male]
+ITEM_20151224_008540	Equipping this will change your hair.
+ITEM_20151224_008541	Dandy Wave Curl
+ITEM_20151224_008542	Curvy Wave Twintail
+ITEM_20151224_008543	Hair [Female]
+ITEM_20151224_008544	Curvy Wave
+ITEM_20151224_008545	Koperon Sword
+ITEM_20151224_008546	Superior Koperon Sword
+ITEM_20151224_008547	Dark Sword
+ITEM_20151224_008548	Soldier's Falchion
+ITEM_20151224_008549	Twin Blade
+ITEM_20151224_008550	Superior Twin Blade
+ITEM_20151224_008551	 - 2% chance to increase magic defense for 15 seconds after receiving magic damage
+ITEM_20151224_008552	Corona Rod
+ITEM_20151224_008553	Superior Corona Rod
+ITEM_20151224_008554	Soldier's Long Rod
+ITEM_20151224_008555	 - Basic attacks have a 1% chance to decrease the enemy's magic defense and increases physical defense for 30 seconds
+ITEM_20151224_008556	 - Attacks have a 5% chance to burn the target for 10 seconds
+ITEM_20151224_008557	Richie Bow
+ITEM_20151224_008558	Superior Richie Bow
+ITEM_20151224_008559	This bow has been used by the hunters for a long time and the use of the bow has been increased after Medzio Diena.
+ITEM_20151224_008560	 - Basic attacks have a 2% chance to poison the target for 20 seconds
+ITEM_20151224_008561	Soldier's Short Bow
+ITEM_20151224_008562	Mark Man
+ITEM_20151224_008563	 - Attacks will decrease enemy's critical hit resistance by {img red_down_arrow 16 16}8 for 20 seconds (Maximum stack of 20)
+ITEM_20151224_008564	Cradu Shooter
+ITEM_20151224_008565	Superior Cradu Shooter
+ITEM_20151224_008566	Burawa
+ITEM_20151224_008567	Superior Burawa
+ITEM_20151224_008568	Soldier's Iron Club
+ITEM_20151224_008569	Iron Fist
+ITEM_20151224_008570	 - Blocking enemy attack will decrease enemy's defense by {img red_down_arrow 16 16}84 for 60 seconds
+ITEM_20151224_008571	This fist shaped mace is said to be created to go against shield wielding enemy. The force of this heavy weapon can shatter a shield.
+ITEM_20151224_008572	Guardman Shield
+ITEM_20151224_008573	Claria Shield
+ITEM_20151224_008574	Ferret Marauder Shield
+ITEM_20151224_008575	Alemeth Tower Shield
+ITEM_20151224_008576	Corseska
+ITEM_20151224_008577	Superior Corseska
+ITEM_20151224_008578	Halberd
+ITEM_20151224_008579	Demon Pike
+ITEM_20151224_008580	Superior Demon Pike
+ITEM_20151224_008581	 - Attacks have a 1% chance to decrease the target's moving speed for 10 seconds
+ITEM_20151224_008582	Even if your assessment is on the line, you can't deny that Ruklys was a great warrior. This spear created during the civil war has many uses.
+ITEM_20151224_008583	Cripple
+ITEM_20151224_008584	 - Attacks have a 33% chance to lower the target's stamina{img red_down_arrow 16 16}1 and activate cripple effect for 30 seconds{nl}  - Stacking cripple effect 10 will lower target's moving speed {img red_down_arrow 16 16}20 for 10 seconds
```

Comment:
- Attacks have a 33% chance to lower a target's stamina {img red_down_arrow 16 16}1 and activate the Cripple Effect for 30 seconds{nl}  - Stacking the Cripple Effect 10 times will lower a target's movement speed by {img red_down_arrow 16 16}20 for 10 seconds